Enhanced storage Use section updates (#3964)
timescale/docs: use-timescale/data-tiering/about-data-tiering.md

        - [services, :serviceId, overview]
---

import TieredStorageBilling from "versionContent/_partials/_tiered-storage-billing.mdx";

# About storage tiers

$COMPANY's tiered storage architecture includes a high-performance storage tier and a low-cost object storage tier. You use the high-performance tier for data that requires quick access, and the object tier for rarely used historical data. Tiering policies move older data asynchronously and periodically from high-performance to low-cost storage, sparing you the need to do it manually. Chunks from a single hypertable, including compressed chunks, can stretch across these two storage tiers. 

## High-performance storage 

> High-performance storage is where your data is stored by default, until you [enable tiered storage][manage-tiering] and start [moving data to the low-cost tier][move-data]. In the high-performance storage, your data is stored in the block format and optimized for frequent querying. The [$HYPERCORE row-columnar storage engine][hypercore] available in this tier is designed specifically for real-time analytics and can compress your data by up to 90%. Coupled with other optimizations, $CLOUD_LONG high-performance storage makes sure your data is always accessible and your queries run at lightning speed. 

Reviewer: ```suggestion
High-performance storage is where your data is stored by default, until you [enable tiered storage][manage-tiering] and [move older data to the low-cost tier][move-data]. In the high-performance storage, your data is stored in the block format and optimized for frequent querying. The [$HYPERCORE row-columnar storage engine][hypercore] is designed specifically for real-time analytics, you can compress data in high-performance storage by up to 90% while improving performance. Coupled with other optimizations, $CLOUD_LONG high-performance storage makes sure your data is always accessible and your queries run at lightning speed. 
```

> $CLOUD_LONG high-performance storage comes in two types—standard and enhanced. 

Reviewer: ```suggestion
$CLOUD_LONG high-performance storage comes in the following types: 
```

> - **Standard** (default): provides up to 16TB of storage and 16,000 IOPS. Based on [AWS EBS gp3][aws-gp3].

Reviewer: ```suggestion
- **Standard**: this default type is based on [AWS EBS gp3][aws-gp3] and provides up to 16TB of storage and 16,000 IOPS.
```

> - **Enhanced**: provides up to 64TB of storage and 64,000 IOPS. Based on [EBS io2][ebs-io2].

Reviewer: ```suggestion
- **Enhanced**: based on [EBS io2][ebs-io2] and provides up to 64TB of storage and 64,000 IOPS.
```

> [See the differences][aws-storage-types] in the underlying AWS storage. You [enable enhanced storage][enable-enhanced] as needed in $CONSOLE. 

Reviewer: Hmmm, I would try and include see the differences in the list.

Author: @rahilsondhi I think you have had a table for this comparison - could you give me access to it, please?

## Low-cost storage

> Once you enable tiered storage, you can start moving rarely used data to the object tier. The object tier is based on AWS S3 and stores your data in the [Apache Parquet][parquet] format. Within a Parquet file, a set of rows is grouped together to form a row group. Within a row group, values for a single column across multiple rows are stored together. The original size of the data in your $SERVICE_SHORT, compressed or uncompressed, does not correspond directly to its size in S3. A compressed hypertable may even take more space in S3 than it does in $CLOUD_LONG.

Reviewer: ```suggestion
Once you enable tiered storage, $CLOUD_LONG moves rarely used data to the object tier. The object tier is based on AWS S3 and stores your data in the [Apache Parquet][parquet] format. Within a Parquet file, a set of rows is grouped together to form a row group. Within a row group, values for a single column across multiple rows are stored together. The original size of the data in your $SERVICE_SHORT, compressed or uncompressed, does not correspond directly to its size in S3. A compressed hypertable may even take more space in S3 than it does in $CLOUD_LONG.
```

Can you link to /enabling-data-tiering/#enable-tiered-storage for enable tiered storage please

Author: This is not technically true. Once you enable tiered storage, you need to set up policies or move data manually. Enabling it does not automatically start the move.

`EXPLAIN` illustrates which chunks are being pulled in from the object storage tier:

1. Fetch data from chunks 42, 43, and 44 from the object storage tier.
1. Skip row groups and limit the fetch to a subset of the offsets in the
   Parquet object that potentially match the query filter. Only fetch the data
   for `device_uuid`, `sensor_id`, and `observed_at` as the query needs only these 3 columns.

The object storage tier is more than an archiving solution. It is also:

- **Cost-effective:** store high volumes of data at a lower cost. You pay only for what you store, with no extra cost for queries.
- **Scalable:** scale past the restrictions of even the enhanced high-performance storage tier.
- **Online:** your data is always there and can be [queried when needed][querying-tiered-data].

> By default, tiered data is not included when querying from a $CLOUD_LONG. However, you can access tiered data by [enabling tiered reads][querying-tiered-data] for a query, a session, or even for all sessions. After you enable tiered reads, when you run regular SQL queries, a behind-the-scenes process transparently pulls data from wherever it's located: the standard high-performance storage tier, the object storage tier, or both.  You can `JOIN` against tiered data, build views, and even define continuous aggregates on it. In fact, because the implementation of continuous aggregates also uses hypertables, they can be tiered to low-cost storage as well.

Reviewer: ```suggestion
By default, tiered data is not included when you query from a $SERVICE_LONG. To access tiered data, you [enable tiered reads][querying-tiered-data] for a query, a session, or even for all sessions. After you enable tiered reads, when you run regular SQL queries, a behind-the-scenes process transparently pulls data from wherever it's located: the standard high-performance storage tier, the object storage tier, or both.  You can `JOIN` against tiered data, build views, and even define continuous aggregates on it. In fact, because the implementation of continuous aggregates also uses hypertables, they can be tiered to low-cost storage as well.
```
